docker-compose.yml

```diff
@@ -7,14 +7,14 @@ services:
       - indico-redis
       - indico-celery
     ports:
-      - "59999"  # tcp socket port
+      - "9090:59999"  # tcp socket port
```

Comment:
I'd use `59999:59999` here

Reply:
But then we would also have to change the `SERVICE_PORT` variable in the config